tools/failures.py

```diff
@@ -80,7 +89,7 @@ def communicate(failures, to_insert, total_detected, testmode, date):
 
 def reset_preseed():
     data = session.query(JobPriorities.expires, JobPriorities.id)\
-                  .filter(JobPriorities.expires != None).all()
+                  .filter(JobPriorities.expires is not None).all()
```

Comment:
It should be fine, however, since I have not tested this part of the code I will delay dealing with this at that point.